kncaputo/rancid-tomatillos
# Use effect

## What's this PR do?
- Add useEffect hook to MovieDetails to render movie correctly based on url id (e.g., /7433027)
- Now movie renders correctly on pushing back button 
- Fixed tests in App.test and MovieDetails.test to accommodate router. Now all pass. 

## Where should the reviewer start?
- 

## How should this be manually tested?
- Start home (MovieContainer view). Click a movie, then click "All movies". Do this ~3 times. 
- Use Back/Forward buttons in browser to navigate through the clicked movies. 
- Movies should render correctly based on id in url

## Any background context you want to provide?
- useEffect in MovieDetails did contain an array of "dependencies" to mock componentDidMount; however...
- this was removed because we do not use the useState hook in MovieDetails. "movie" is simply passed as a prop from App

## Questions:
- N/A
